com.github.hneemann.digital/com.github.hneemann.digital.json

```diff
@@ -0,0 +1,123 @@
+{
+  "app-id": "com.github.hneemann.digital",
+  "branch": "0.31",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "23.08",
+  "sdk": "org.freedesktop.Sdk",
+  "sdk-extensions": [
+      "org.freedesktop.Sdk.Extension.openjdk21"
+  ],
+  "command": "Digital.sh",
+  "finish-args": [
+      "--persist=.java",
+      "--share=ipc",
+      "--share=network",
+      "--socket=x11",
+      "--socket=wayland",
```

Comment:
It's either x11 or fallback-x11 + wayland not both. See the note in https://docs.flatpak.org/en/latest/sandbox-permissions.html#standard-permissions